view.php

```diff
@@ -104,9 +104,9 @@
       . "<a href='$wwsetlink'>" . get_string("iframeNoShow-2", "wwassignment")
       ."</a><p align='center'></iframe></p>\n"
       );
-print("<iframe id='wwPage' src='$wwsetlink' frameborder='1' "
+print("<iframe id='wwPage' src='$wwsetlink' frameborder='0' "
       . "width='".$CFG->wwassignment_iframewidth."' "
-      . "height='".$CFG->wwassignment_iframeheight."'>"
+      . "height='".$CFG->wwassignment_iframeheight."' scrolling='no' >"
```

Comment:
https://devel3.webwork.rochester.edu/moodle/mod/wwassignment/view.php?id=1022  
(Linear Algebra and ODE, Spring 2009 -- the homework set is WebWork Orientation -- problem 1
(on this site I have reset the iframe to allow scrolling).    

There are also other problems, "scaffolding problems" which can be very long  https://hosted2.webwork.rochester.edu/webwork2/2014_HKUST_demo/SequentialWordProblem/1/.

Finally whether scrolling is needed or not may depend a great deal on what device you are using to view the problem.  I think it is safer to allow scrolling.  I don't see any particular downside to having the option available.